doc/MANUAL.md

```diff
@@ -440,6 +448,46 @@ extract = true
 
 Although the examples in this section show results for a single library to make them shorter and easier to read, the tool was designed to be run on entire directories and will scan all files in a directory if one is supplied.
 
+### -a INTERMEDIATE_PATH, --append INTERMEDIATE_PATH
+
+This option allows you to save the output in form of an intermediate report which you can later merge with other reports of the same format.  
```

Comment:
Let's put a link here to the how-to guide you made so that people know they can click through to get more information.